appcues/src/main/java/com/appcues/statemachine/states/RenderingStepState.kt

```diff
@@ -31,6 +33,9 @@ internal data class RenderingStepState(
             } else {
                 toEndingStep(action)
             }
+            is Reset -> {
+                toBeginningStep()
+            }
```

Comment:
seems ok to use existing `Reset` here to me. The PR description has some older info about `RestartStep`. Would be good to update and clean that up.

Maybe in the future it would use `Retry` from [here](https://github.com/appcues/appcues-android-sdk/pull/528/files#diff-a4ce2dcad8b214bb7929fd1f82a3c39b079064cd08a542c9fa209c5b8df0c3c4R19)? (or vice versa, not sure)

Reply:
yeah we ok use Retry in the future, both naming here could be confusing for whats actually doing